server/server.go

```diff
@@ -508,7 +511,9 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		WorkingDir:          workingDir,
 		Webhooks:            webhooksManager,
 		WorkingDirLocker:    workingDirLocker,
-		TerraformOutputChan: terraformOutputChan,
+		ProjectCmdOutputHandler: handlers.DefaultProjectCommandOutputHandler{
```

Comment:
why are you defining a new `handlers.DefaultProjectCommandOutputHandler`? You already created it pass in `projectCmdOutputHandler`